development/tools/cmd/githubstats/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/jamiealquiza/envy"
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+	"github.com/spf13/cobra"
+	"golang.org/x/oauth2"
+)
+
+type Issues struct {
+	Open   IssuesOpen
+	Closed IssuesClosed
+}
+
+type IssuesOpen struct {
+	TotalCount       int64
+	Bugs             int64
+	PriorityCritical int64
+	Regressions      int64
+	TestFailing      int64
+	TestMissing      int64
+}
+
+type IssuesClosed struct {
+	TotalCount       int64
+	Bugs             int64
+	PriorityCritical int64
+	Regressions      int64
+	TestFailing      int64
+	TestMissing      int64
+}
+
+type Report struct {
+	Issues     Issues
+	Type       string
+	Owner      string
+	Repository string
+	Timestamp  time.Time
+}
+
+type Config struct {
+	GithubAccessToken string
+	GithubRepoOwner   string
+	GithubRepoName    string
+}
+
+var (
+	query struct {
+		Repository struct {
+			Issues struct {
+				TotalCount int64
+			} `graphql:"issues(states: [$state], labels: $labels)"`
+		} `graphql:"repository(owner: $repositoryOwner, name: $repositoryName)"`
+	}
+
+	query_no_labels struct {
+		Repository struct {
+			Issues struct {
+				TotalCount int64
+			} `graphql:"issues(states: [$state])"`
+		} `graphql:"repository(owner: $repositoryOwner, name: $repositoryName)"`
+	}
+)
+
+func getStats(cfg Config) {
+	src := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: cfg.GithubAccessToken},
+	)
+	httpClient := oauth2.NewClient(context.Background(), src)
+
+	client := githubv4.NewClient(httpClient)
+
+	r := Report{
+		Timestamp:  time.Now(),
+		Type:       "GithubIssuesStatsReport",
+		Repository: cfg.GithubRepoName,
+		Owner:      cfg.GithubRepoOwner,
+	}
+
+	variables := map[string]interface{}{
+		"repositoryOwner": githubv4.String(cfg.GithubRepoOwner),
+		"repositoryName":  githubv4.String(cfg.GithubRepoName),
+	}
+
+	// Total count
+	variables["state"] = githubv4.IssueState("OPEN")
+	err := client.Query(context.Background(), &query_no_labels, variables)
+	if err != nil {
```

Comment:
simply use fatalOnError:)

```suggestion
	fatalOnError(err)
```